src/main/java/com/aws/greengrass/mqtt/bridge/model/Mqtt5RouteOptions.java

```diff
@@ -17,7 +17,7 @@
 public class Mqtt5RouteOptions {
 
     public static final boolean DEFAULT_NO_LOCAL = false;
-    public static final boolean DEFAULT_RETAIN_AS_PUBLISHED = true;
+    public static final boolean DEFAULT_RETAIN_AS_PUBLISHED = false;
```

Comment:
why are you changing the default? Use the default that we decided in the design document.

Reply:
Discussed this with Joe, we think that the default was wrong the first time around. V3 doesn't support retain as published, so the default value should have been false to maintain backwards compatibility. I think the design doc may have been talking about keeping the flag true for the local client